service/controller/app.go

```diff
@@ -91,6 +107,7 @@ func newAppResources(config AppConfig) ([]resource.Interface, error) {
 	}
 
 	resources := []resource.Interface{
+		configversionResource,
```

Comment:
Even though it doesn't make much sense now I'd move it to a separate controller (app_catalog_entry.go) to make our life easier when we are unblocked with https://github.com/giantswarm/giantswarm/issues/14832.